### ISSUE TITLE
Breaking grass or tall grass with AIOT Botania's Shears drops random seeds instead of grass

### ISSUE THREAD

**Issue by vitaeum** [MelanX/aiotbotania#71]
**Versions**

Minecraft: 1.16.5
Forge: 36.2.26
Botania: 1.16.5-420
AIOT Botania: 1.8.4
Modpack: No other mods

**Description**

When breaking grass, tall grass, or ferns with AIOT Botania shears, a random seed is dropped instead of the respective block. Exactly one seed always drops, and it is randomly picked from any item with the forge seeds tag. This problem exists with no other mods, but is still present when other mods are added, and modded seeds can also drop. Unusually, this bug also prevents Manasteel or Elementium Shears from functioning on grass, but they do not also drop a random seed. All shears I have named still work on any other shearable block, and can still shear sheep and pumpkins.

**How to reproduce**

1. Break grass with AIOT Botania shears.
2. It drops a random item tagged as a seed instead of grass.

**Expected behavior**

1. Break grass with AIOT Botania shears.
2. It drops the grass block and nothing else.

**Comment by MelanX** [comment]
This bug is an intended feature. You need to enchant the shears with silk touch to not drop a seed. I could add a config for that but wouldn’t change the default behavior. Additionally, I could add that information to the lexicon.

**Comment by kezmodius** [comment]
This very much DOES need a config. My 1.20.1 modpack is attempting to severely limit Mystical Agriculture seeds and players discovered that using the livingwood and livingrock shears drops these seeds at random, including tier 4 and 5 ones. In addition, it is dropping seeds I have configured out of the pack entirely with Mystical Customizations. I have removed the tools in question for now, but would really appreciate this seed drop to be configurable.
